src/classes/AuthResponse.php

```diff
@@ -36,6 +36,10 @@ class AuthResponse
 
     public ?string $mfaSecret = null;
 
+    public ?bool $isAdmin = null;
```

Comment:
why is it nullable? Can't we have the default value as `false`. I always prefer to have non nullable types if possible.